### DISCUSSION TITLE
Labels in Pies/Gauges of WPFSample are not in the correct place

### DISCUSSION THREAD

**Discussion by MrZRyan** [Live-Charts/LiveCharts2#1133] [Q&A]
Hello, I was testing the WPFSample and I have seen that there is something wrong in LabelsPosition of GaugeBuilder. 
The page is Pies/Gauges

![2023-07-25 081036](https://github.com/beto-rodriguez/LiveCharts2/assets/1562810/6968df39-154b-4c23-a86d-390817ce42e1)

**Comment by beto-rodriguez** [comment]
Hi and thanks for the report, I will move this to an issue.
